spec/graphql/language/lexer_spec.rb

```diff
@@ -85,7 +92,7 @@
       assert_equal 8, str_token.col
       assert_equal '(STRING "c" [1:8])', str_token.inspect
       rparen_token = tokens[6]
-      assert_equal '(RPAREN ")" [1:10])', rparen_token.inspect
+      assert_equal '(RPAREN ")" [1:11])', rparen_token.inspect
```

Comment:
Comment above: I think we were off by 1 all this time?

Reply:
Yes, I believe it 😆 Even here in the "calculates string position correctly" test.  I think we also used to miscalculate the _beginning_ position, but obviously I didn't think to consider the end position too.